lib/Extension/LanguageServerPhpCsFixer/Model/PhpCsFixerProcess.php

```diff
@@ -135,7 +135,7 @@ public function run(array $env, string ...$args): Promise
      */
     private function resolveExtraArgs(?SemVersion $version, string ...$args): array
     {
-        if ($version?->greaterThanOrEqualTo(new SemVersion('3.89.2'))) {
+        if ($version?->greaterThanOrEqualTo(SemVersion::fromString('3.89.2'))) {
```

Comment:
made the constructor private (to avoid exposing the internal state of the object - f.e. in this case it may make sense to refactor later to have distinct sem ver components rather than a single string).